compiler/hash-semantics/src/storage/cache.rs

```diff
@@ -61,12 +61,6 @@ impl<K: Copy + Hash + Eq, V: Clone> PartialStore<K, V> for CacheStore<K, V> {
     fn internal_data(&self) -> &RefCell<HashMap<K, V>> {
         self.store.internal_data()
     }
-
-    /// Clear the [CacheStore] and metrics.
-    fn clear(&self) {
-        self.store.clear();
-        self.reset_metrics();
-    }
```

Comment:
Was this does not being used before to reset the cache after each module was being walked?

Reply:
Idk why that was deleted, added it back.